torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -120,6 +121,11 @@ struct ConstantPropagator {
     for (size_t i = 0; i < outputs.size(); ++i) {
       auto new_output = tryInsertConstant(*graph, outputs[i]);
       if (new_output) {
+        GRAPH_UPDATE(
+            "Folding\n",
+            n->outputs()[i]->debugName(),
```

Comment:
Nit: might be worth adding a '`%`' before the value name to make it clear we're talking about a value here.

Reply:
ok